src/docs/product/integrations/integration-platform/public-integration.mdx

```diff
@@ -0,0 +1,210 @@
+        'client_id': 'your-client-id',
+        'client_secret': 'your-client-secret',
+    }
+
+    resp = requests.post(url, json=payload)
+    data = resp.json()
+
+    new_token = data['token']
+    new_refresh_token = data['refreshToken']
+    # ... Securely update the token and refresh_token in DB...
+
+    return new_token
+```
+
+Instead of keeping track of times and passively refreshing when a token expires, another way you can handle refreshing tokens is to actively capture exceptions raised by requests that receive a 401 unauthorized response from Sentry, refresh the token, and remake the request.
+
+### Verifying Installations (optional)
+
+Typically, if you have the redirect URL configured, there is work happening on your end to "finalize" the installation. If this is the case, we recommend enabling the "Verify Install" option for your integration. Once enabled, you'll need to send a request marking the installation as officially "installed":
+
+```python
+requests.put(
+    u'https://sentry.io/api/0/sentry-app-installations/{}/'.format(install_id),
+    json={'status': 'installed'},
+)
+```
+
+
+## Uninstallation
+
+When a user uninstalls your integration, you'll receive a webhook request to your webhook URL.
+
+## Integration Webhooks
+
+In addition to the [un]installation webhook requests, all of the webhooks that you selected when configuring your integration will be routed to your webhook URL.
+
+Continuing from our example, here we're implementing the webhook URL endpoint, `/webhook`. In this case, that includes when an issue is created. In this method, you'll use the Sentry API to check if the issue belongs to a project called "Backend", and if it does, you'll forward the issue to Pushover:
+
+```python
+@app.route('/webhook', methods=['POST'])
+def webhook():
+    data = json.loads(request.data)
+
+    if data['action'] != 'created':
+        return
+
+    issue_id = data['data']['issue']['id']
+    install_id = data['installation']['uuid']
+
+    issue_details = get_sentry_issue(install_id, issue_id)
+
+    if issue_details['project']['name'] != 'Backend':
+        return
+
+    event = data['data']['event']
+
+    payload = {
+        'user': 'pushover-user-key',
+        'token': 'pushover-api-token',
+        'message': event['message'][:1024],
+        'title': event['message'][:250],
+        'url': event['url'],
+        'url_title': 'Issue Details',
+        'priority': 0,
+    }
+    requests.post('https://api.pushover.net/1/messages.json', data=payload)
+
+def get_sentry_issue(install_id, issue_id):
+    token = retrieve_from_db(install_id)
+
+    url = u'https://sentry.io/api/0/issues/{}/'.format(issue_id)
+    headers = {'Authorization': u'Bearer {}'.format(token)}
+
+    resp = requests.get(url, headers=headers)
+    return resp.json()
+```
+
+For more information, check out the [full documentation on Webhooks](/product/integrations/integration-platform/webhooks/).
+
+## Published State
```

Comment:
I think this heading can be changed to 'Publishing' since that'd match what you reference at the top of the page. I also think this would be a good section to include the image requirements for publishing, maybe as an H3 underneath here.